bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import discord
+import requests
```

Comment:
Since the discord.py bot framework is asynchronous, the requests library cannot be used. You can check out other cogs in this repository for examples on how to use aiohttp to make asynchronous web requests.